engine/overworld/doors.asm

```diff
@@ -25,33 +25,21 @@ IsPlayerStandingOnDoorTile: ; 1a609 (6:6609)
 	and a
 	ret
 
-DoorTileIDPointers: ; 1a62c (6:662c)
-	db OVERWORLD
-	dw OverworldDoorTileIDs
-	db FOREST
-	dw ForestDoorTileIDs
-	db MART
-	dw MartDoorTileIDs
-	db HOUSE
-	dw HouseDoorTileIDs
-	db FOREST_GATE
-	dw TilesetMuseumDoorTileIDs
-	db MUSEUM
-	dw TilesetMuseumDoorTileIDs
-	db GATE
-	dw TilesetMuseumDoorTileIDs
-	db SHIP
-	dw ShipDoorTileIDs
-	db LOBBY
-	dw LobbyDoorTileIDs
-	db MANSION
-	dw MansionDoorTileIDs
-	db LAB
-	dw LabDoorTileIDs
-	db FACILITY
-	dw FacilityDoorTileIDs
-	db PLATEAU
-	dw PlateauDoorTileIDs
+DoorTileIDPointers: ; 1a7a8 (6:67a8)
```

Comment:
We want to keep pokered and pokeyellow in sync as much as possible, so if you make edits like this, could you do the same in pokered?